pango/src/auto/tab_array.rs

```diff
@@ -92,4 +134,37 @@ impl TabArray {
             );
         }
     }
+
+    #[cfg(any(feature = "v1_50", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v1_50")))]
+    #[doc(alias = "pango_tab_array_sort")]
+    pub fn sort(&mut self) {
+        unsafe {
+            ffi::pango_tab_array_sort(self.to_glib_none_mut().0);
+        }
+    }
+
+    #[cfg(any(feature = "v1_50", feature = "dox"))]
```

Comment:
those functions probably shouldn't take mut self, but all the functions in this type seems to be doing so already so... I will leave cleaning that up to someone that needs it/knows how it should be used

Reply:
Yes the getters at least shouldn't. Please create an issue :)

Reply:
It seems to break the build, let me fix that :)